Octokit/Models/Request/PullRequestUpdate.cs

```diff
@@ -29,6 +29,11 @@ public class PullRequestUpdate
         /// </summary>
         public string Base { get; set; }
 
+        /// <summary>
+        /// Whether maintainers of the base repository can push to the pull request's HEAD if it is a branch
+        /// </summary>
+        public bool MaintainerCanModify { get; set; }
```

Comment:
This should also be nullable `bool?` so that the user is able to not specify a value and have it not change this property on the PR they are updating